README.md

```diff
@@ -148,32 +156,64 @@ sbt packArchiveZip # creates a single zip with resources
 This will use the sbt-pack plugin to create a JAR file and some helper scripts to run the server. The output for this will be in the `PSF-LoginServer\target` directory. Now you can copy the ZIP file to a server you want to run it on. You will need the Java 8 runtime (JRE only) on the target to run this. In the ZIP file, there is a `bin/` directory with some helper scripts. Run the correct file for your platform (.BAT for Windows and shell script for Unix).
 
 ### Generating Documentation
+
 Using SBT, you can generate documentation for both the common and pslogin projects using `sbt unidoc`.
 
 Current documentation is available at [https://psforever.github.io/docs/master/index.html](https://psforever.github.io/docs/master/index.html)
 
 ## Troubleshooting
 
 #### Unable to initialize pscrypto
+
 If you get an error like below
+
 ```
 12:17:28.037 [main] ERROR PsLogin - Unable to initialize pscrypto
 java.lang.UnsatisfiedLinkError: Unable to load library 'pscrypto': Native library (win32-x86-64/pscrypto.dll) not found in resource path
 ```
+
 Then you are missing the native library required to provide cryptographic functions to the login server. To fix this, you need a binary build of [PSCrypto](#downloading-pscrypto).
 
 If you are still having trouble on Linux, try putting the library in `root directory/pscrypto-lib/libpscrypto.so`.
 
 ## Contributing
+
 Please fork the project and provide a pull request to contribute code. Coding guidelines and contribution checklists coming soon.
 
 ## Get in touch
 
-* Website: http://psforever.net
-* Discord (chat with us): https://discord.gg/0nRe5TNbTYoUruA4
+- Website: http://psforever.net
+- Discord (chat with us): https://discord.gg/0nRe5TNbTYoUruA4
   - Join the #code channel and ask any questions you have there
 
 Chord is the lead developer and you can contact him on Discord as Chord or by email [chord@tuta.io](mailto:chord@tuta.io). Discord is preferred.
 
+## Tools
+
+### decodePackets
+
+The decodePackets program can be used to decode GameLogger `.gcap` packet captures. Requires
+[gcapy](https://github.com/psforever/gcapy) to run.
```

Comment:
Probably worth updating this with a quick note about preprocessed files, and a couple examples of the new flags for how to run it below?